spec/definitions/TokenInfoV1.yaml

```diff
@@ -0,0 +1,63 @@
+required:
+  - id
+  - enterprise
+type: object
+properties:
+  id:
+    title: Id
+    description: Token id
+    type: string
+    format: integer
+    readOnly: true
+  enterprise:
+    title: Enterprise id
+    description: Id of the enterprise resource
+    type: string
+    format: uuid
+    readOnly: true
+  user:
+    title: User id
+    description: Id of the user resource
+    type: string
+    readOnly: true
+  developer_app:
+    title: Developer app id
+    description: Id of the developer app resource that you created using the esper dev console
+    type: string
+    format: uuid
+    readOnly: true
+  source_refresh_token:
+    title: Refresh token
+    description: The refresh token is used to refresh your expired API token
+    type: string
+    readOnly: true
+  token:
```

Comment:
Specify what type token and length of token.

Reply:
what will be the use of length of the token ?